code/modules/reagents/chemistry/reagents/drinks/alcohol_reagents.dm

```diff
@@ -107,6 +109,19 @@
 		for(var/datum/surgery/surgery as anything in exposed_mob.surgeries)
 			surgery.speed_modifier = max(0.1 * power_multiplier, surgery.speed_modifier)
 
+/datum/reagent/consumable/ethanol/proc/alcohol_effectiveness() // for decrease in power of healing after time
```

Comment:
```suggestion
/// Returns a multiplier (0-1) which can be used to scale effectiveness of the reagent depending on how long it's been since creation
/datum/reagent/consumable/ethanol/proc/alcohol_effectiveness() 
```